### DISCUSSION TITLE
Poetry vs npm

### DISCUSSION THREAD

**Discussion by alekslario** [python-poetry/poetry#3540] [Q&A] [answered]
I'm a little bit confused. Does this lib actually install dependencies like npm does? Or I have to install with pip? Also it doesn't raise  errors when I `poetry build` with no dependencies actually presently installed locally or globally. :confused: I am missing something? What is the optimal workflow to manage dependencies? 
Edit:
 It seems that `poetry add package` does install something but not locally. Where is the default installation location and how to make it local? Got `poetry.toml` with 
```
[virtualenvs]
create = true
in-project = true
```
but that doesn't help.

**Comment by alekslario** [comment, marked as answer]
I figured it out :joy:
Edit:
My solution - Run `poetry env use python` in the project directory. Given that you have a `poetry.toml` file, `.venv` folder will be created in the same directory. That's where all installations will go.

**Reply by sinoroc** [reply]
Since it's a "_Q & A_" section, would you mind sharing your actual answer and mark it as "accepted"? For those who might have the same question. Otherwise this question is still marked as "unanswered"...

**Reply by alekslario** [reply]
Thanks for the tip.
